### PR TITLE
Adjust field size

### LINKED ISSUE #268: Postgrex.Error: ERROR 22001 (string_data_right_truncation) value too long for type character varying(255)
Sentry Issue: [BACKEND-81](https://sentry.io/organizations/cambiatus/issues/3466878312/?referrer=github_integration)

```
Postgrex.Error: ERROR 22001 (string_data_right_truncation) value too long for type character varying(255)
  File "lib/ecto/adapters/sql.ex", line 932, in Ecto.Adapters.SQL.raise_sql_call_error/1
  File "lib/ecto/repo/schema.ex", line 744, in Ecto.Repo.Schema.apply/4
  File "lib/ecto/repo/schema.ex", line 367, in anonymous fn/15 in Ecto.Repo.Schema.do_insert/4
  File "lib/cambiatus_web/resolvers/accounts.ex", line 81, in CambiatusWeb.Resolvers.Accounts.sign_in/3
    Cambiatus.Auth.create_session(params)
  File "lib/absinthe/resolution.ex", line 209, in Absinthe.Resolution.call/2
...
(3 additional frame(s) were not displayed)

(Postgrex.Error ERROR 22001 (string_data_right_truncation) value too long for type character varying(255))
```

### PR DESCRIPTION
**Pull request by lucca65**
## What issue does this PR close
Closes #268

## Changes Proposed ( a list of new changes introduced by this PR)
- Adjust field on auth_sessions table to be text instead of varchar(255)

## How to test ( a list of instructions on how to test this PR)
N/A

### CODE REVIEW

File: priv/repo/migrations/20220806190533_session_token_as_text.exs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+defmodule Cambiatus.Repo.Migrations.SessionTokenAsText do
+  use Ecto.Migration
+
+  def change do
+    alter table(:auth_sessions) do
+      modify(:token, :text, null: false, comment: "Markdown description of the category")
```
Comment (NeoVier) [inline priv/repo/migrations/20220806190533_session_token_as_text.exs:6]: > comment: "Markdown description of the category"

I don't think that's the right comment 😆

**Review by NeoVier — CHANGES_REQUESTED** [review]

File: priv/repo/migrations/20220806190533_session_token_as_text.exs (hunk 1 above)
Comment (lucca65) [reply priv/repo/migrations/20220806190533_session_token_as_text.exs]: oh, glad you caught this one bro! 😥

**Review by NeoVier — APPROVED** [review]
🧹

**Review by MatheusBuss — APPROVED** [review]
:+1:

**Pull request merged**